RGBMatrixEmulator/adapters/browser_adapter/static/assets/client.js

```diff
@@ -15,38 +14,11 @@ function init() {
     let requestTimeSmoothing = 0.2;    // larger=more smoothing
     let targetTime = 1000 / fpsTarget;
 
-    let retryCount = 0;
     let socket = generateSocket();
 
-    function requestImage() { 
+    function requestImage() {
         requestStartTime = performance.now();
-
-        waitForSocketReady(socket, function () {
-            socket.send('more');
-        });
-    }
-
-    function waitForSocketReady(socket, callback) {
```

Comment:
We no longer need to check for socket status as once we get an error we recreate the connection.

However this may not be desirable for allowing a slow websocket server to recover. (it will always be reconnected).